runtime/j9vm/javanextvmi.cpp

```diff
@@ -571,7 +571,15 @@ JVM_GetClassFileVersion(JNIEnv *env, jclass cls)
 JNIEXPORT void JNICALL
 JVM_VirtualThreadHideFrames(JNIEnv *env, jobject vthread, jboolean hide)
 {
-	/* TODO toggle hiding of stack frames for JVMTI */
+	J9VMThread *currentThread = (J9VMThread *)env;
+	j9object_t threadObject = J9_JNI_UNWRAP_REFERENCE(vthread);
+	j9object_t carrierThread = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CARRIERTHREAD(currentThread, threadObject);
+	J9VMThread *targetThread = J9VMJAVALANGTHREAD_THREADREF(currentThread, carrierThread);
+	if (JNI_TRUE == hide) {
```

Comment:
I think this should only compare with `JNI_FALSE`, everything else should be considered "true".

Reply:
Or better yet don't do comparisons on things that are booleans (even though this is a fake kind of boolean).

Reply:
Nothing changed here. It should be
```c
	if (hide) {
```